--- origsrc/libXmu-1.1.2/src/RdBitF.c	2013-09-07 23:28:18.000000000 -0500
+++ src/libXmu-1.1.2/src/RdBitF.c	2013-11-21 18:32:08.672055400 -0600
@@ -56,6 +56,7 @@ in this Software without prior written a
 #include <X11/Xmu/Drawing.h>
 #ifdef WIN32
 #include <X11/Xwindows.h>
+#include <direct.h>
 #endif
 
 #define MAX_SIZE 255
--- origsrc/libXmu-1.1.2/src/StrToCurs.c	2013-09-07 23:28:18.000000000 -0500
+++ src/libXmu-1.1.2/src/StrToCurs.c	2013-11-21 18:39:39.917865200 -0600
@@ -70,6 +70,7 @@ SOFTWARE.
 #endif /* X_NOT_POSIX */
 #ifndef PATH_MAX
 #ifdef WIN32
+#include <stdlib.h>
 #define PATH_MAX 512
 #else
 #include <sys/param.h>
